src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
+    private String color;
+    private double radius;
+
+    public Circle(String color, double radius) {
+        this.color = color;
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
```

Comment:
Avoid using direct calls to `System.out.println()` in your class methods for drawing. Consider using an interface with a method like `draw()` that can be implemented to handle drawing in a more flexible way, such as returning a String representation of the figure.